public/app/features/provisioning/GettingStarted/GettingStarted.tsx

```diff
@@ -18,7 +18,6 @@ const featureIni = `# In your custom.ini file
 
 [feature_toggles]
 provisioning = true
-kubernetesClientDashboardsFolders = true
```

Comment:
^^^ enabled by default not

Reply:
@ryantxu should we remove the recommendation for `grafanaAPIServerEnsureKubectlAccess`?

Reply:
doesnt seem to serve any purpose